nodeJS/express/installing_postgresql.md

```diff
@@ -0,0 +1,295 @@
+<!-- markdownlint-disable MD024 TOP004 -->
+
+### Introduction
+
+Data persistence is integral to the vast majority of web applications. Data persistence is achieved through databases. Being able to understand how to structure, build, and query your own database are important skills for any full-stack developer to have.
+
+For example, consider The Odin Project. Where is your username stored? Where are project submissions stored? Where does this lesson content, the one you're reading right now, stored? Yes, a database!
+
+We've chosen [PostgreSQL](https://www.postgresql.org/) as our database of choice for the curriculum. All SQL knowledge that you've learned in the earlier course should be transferable to PostgreSQL. We'll also learn new PostgreSQl specific things. Watch through this [fun and short introduction video to PostgreSQL by Fireship](https://www.youtube.com/watch?v=n2Fluyr3lbc).
+
+<div class="lesson-note lesson-note--critical" markdown="1">
+
+#### Prerequisite SQL knowledge
+
+Make sure you've completed the [SQL course](https://www.theodinproject.com/paths/full-stack-javascript/courses/databases).
+
+This lesson and all subsequent lessons will assume you understand SQL syntax and concepts.
+
+</div>
+
+In this lesson, we'll install PostgreSQl and in the next lesson we'll learn how to use it in our Express application.
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- How to install PostgreSQL.
+
+### Installing PostgreSQL
+
+If you've been following the curriculum, you should already have installed all the supporting bits and pieces. If not, please go back and refer to [those lessons](https://www.theodinproject.com/guides/installations) before continuing.
+
+Pick your operating system below to get the appropriate steps for installing PostgreSQL.
+
+<details markdown="block">
+
+<summary class="dropDown-header">Linux</summary>
+
+### Step 1: Make sure the system is up to date
+
+Before installing PostgreSQL, it's a good idea to make sure the operating system is up to date. To update our system, run this command:
+
+```bash
+sudo apt update && sudo apt upgrade
+```
+
+### Step 2: Install the PostgreSQL packages
+
+After our system is up to date, we will install the packages for PostgreSQL.
+
+```bash
+sudo apt install postgresql postgresql-contrib libpq-dev
+```
+
+After installation is complete, let's start the server using this command:
+
+```bash
+sudo systemctl start postgresql.service
+```
+
+<div class="lesson-note lesson-note--warning" markdown="1">
+
+#### Systemctl and WSL2
+
+Systemctl is not supported on WSL2, and the above command won't work. Instead, run the following command:
+
+```bash
+sudo service postgresql start
+```
+
+</div>
+
+Got an error, or don't see an active service? Come visit the [Discord](https://discord.gg/V75WSQG) for some help!
```

Comment:
Same thing for line 196 in the MacOS instructions.
Planned descriptive link change to match the Rails version when that gets sorted.

```suggestion
Got an error, or don't see an active service? Come visit [our Discord server](https://discord.gg/V75WSQG) for some help!
```